packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -481,11 +508,13 @@ class ChartController extends DisposableController
   /// visible.
   double timestampXCanvasCoord(int timestamp) {
     final index = timestamps.indexOf(timestamp);
-    final visibleIndex = normalizeTimestampIndex(index);
-    if (visibleIndex >= 0) {
-      return (visibleIndex * tickWidth).toDouble();
-    } else {
-      return -1;
+    if (index >= 0) {
+      // Valid index.
+      final visibleIndex = normalizeTimestampIndex(index);
+      if (visibleIndex >= 0) {
+        return (visibleIndex * tickWidth).toDouble();
+      }
     }
+    return -1;
```

Comment:
-1.0 seems like a surprising value. Would returning null make more sense?

Reply:
Sure I'll do that.